spyder/plugins/editor/widgets/completion.py

```diff
@@ -277,14 +277,30 @@ def hide(self):
         QListWidget.hide(self)
         QToolTip.hideText()
 
+    def event(self, event):
+        """Override Qt method to process tab keypress."""
```

Comment:
The `Tab` key is not actually propagated to he keypress so I had to filter it here